gocd/templates/clickhouse-query-comparer.jsonnet

```diff
@@ -0,0 +1,67 @@
+local gocdtasks = import 'github.com/getsentry/gocd-jsonnet/libs/gocd-tasks.libsonnet';
+
+local pipeline_group = 'snuba';
+local pipeline_name = 'clickhouse-query-fetcher';
+
+local generate_compare_job() =
```

Comment:
maybe this should be something like `generate_query_job()` since it no longer just compare.

Reply:
I think whenever we make this generic we can, but this file is still specific for the comparer for now.